_policies/04_compare_simulations.md

```diff
@@ -20,6 +20,10 @@ the same programs (or potentially additional programs).
 * Are best practices being used in the simulation comparisons?
 * Are the methods applied to systems which for which reliable independent data are available?  Are the systems considered of sufficient complexity that readers will find the results convincing and generalizable?
 * Have the authors addressed whether a naive user could expect similar performance and why or why not?
+* Do the authors address differences in the underlying philosophy of different simulation programs?
```

Comment:
@dwsideriusNIST - can you give an example of how this would be relevant? I'm not sure I'm getting what you're driving at here.

Reply:
Honestly, I'm not entirely sure what I am driving at here.

An quick thought (sorry, head cold has me slow today) is whether a sim is really trying to sample phase space properly. Like an MD program that applies a non-physical thermostat. It might give the right averages, but wrong fluctuations.

For now, I'm going to cut this line and maybe add it back later. I'm happy to continue the discussion. Perhaps there was something more profound that I intended when I wrote this yesterday.

Reply:
Haha, sounds good. Past self has outsmarted present self. Feel better soon.